solidity/deploy/41_transfer_redemption_watchtower_ownership.ts

```diff
@@ -0,0 +1,19 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
> Where do we set the Token Holder DAO as the watchtower manager?

This will be done through the `enableWatchtower` call after deployment.

> Also, is the newly deployed watchtower proxy attached to the already existing ProxyAdmin instance?

Yes, this is the plan.